lib/uknetdev/include/uk/netbuf.h

```diff
@@ -414,6 +421,21 @@ void uk_netbuf_connect(struct uk_netbuf *headtail,
 void uk_netbuf_append(struct uk_netbuf *head,
 		      struct uk_netbuf *tail);
 
+#if CONFIG_LIBUKSGLIST
+/**
+ * This function creates a scatter gather list from the netbuf.
```

Comment:
`This function appends a netbuf to a scatter gather list`?

Reply:
Right, I did not write this comment, I simply moved the one from before. Will re-write it.